packages/example-app-base/src/utilities/getQueryParam.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Gets a query parameter value
+ * @param name Query parameter name
+ * @param url URL (defaults to window.location.href)
+ */
+export function getQueryParam(name: string, url?: string): string | null {
```

Comment:
You should probably use the `url` package

Reply:
Why? We only need this small part of it, and a spec-compliant URL parsing implementation won't handle funky hash route+query URLs anyway